doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -1682,39 +1695,92 @@ PSI Variables by Alpha
    by 1.4 opposite-spin and 0 same-spin contributions, with
    any singles carried along.
 
-.. psivar:: TDDFT ROOT 0 -> ROOT m EXCITATION ENERGY - h SYMMETRY
-   TD-fctl ROOT 0 -> ROOT m EXCITATION ENERGY - h SYMMETRY
-   ADC ROOT 0 -> ROOT m EXCITATION ENERGY - h SYMMETRY
-   EOM-CCSD ROOT 0 -> ROOT m EXCITATION ENERGY - h SYMMETRY
+.. psivar:: TD-fctl ROOT 0 -> ROOT m EXCITATION ENERGY
+   TD-fctl ROOT 0 (h) -> ROOT m (i) EXCITATION ENERGY
+   TD-fctl ROOT 0 -> ROOT m EXCITATION ENERGY - h TRANSITION
 
-   The excitation energy of given method from ground state to root m
-   in h symmetry (if available). DFT functional labeled if canonical.
+   The excitation energy of a given method from ground state to root m.
+   DFT functional labeled if canonical.
+   m in the first and third variables is the index of the root among all roots.
+   m in the second variable is the index of the root among all roots of irrep i.
+   h, i in the second variable are the irreps of the roots.
+   h in the third variable is the irrep of the transition between roots.
```

Comment:
I'm thinking for energies at least (as opposed to properties: dipole, oscillator, rotary, etc.) these entries might be easier to read if only the method was changing. For example, adc/tddft/eomcc no-sym excitation as one glossary entry. adc/tddft/eom per-sym excitation as another, and transition-sym as another. If you agree but don't want to rearrange it all now, I can do a pass at the end of the qcvars series. Glad of other opinions.

Reply:
There are advantages either way, and I have no strong opinions on which of the two we take.

But if we shift to your suggestion, I have two requests:
* The "see X for indexing convention" line changes to "see the nth access call in `Notes on Psivars`" or whatever the appropriate sphinx-ification is.
* If we break up one variable like that, we break up _all_ variables like that.